migrations/20241030234724_first_migration.sql

```diff
@@ -0,0 +1,21 @@
+-- +goose Up
+-- create "metrics" table
+CREATE TABLE "metrics" (
+  "id" character varying(255) NOT NULL,
+  "created_at" timestamptz NULL,
+  "updated_at" timestamptz NULL,
+  "deleted_at" timestamptz NULL,
+  "type" text NOT NULL,
+  "m_source" text NULL,
```

Comment:
[CHANGE REQUEST] Please find the suitable name

Reply:
I have removed it in a second migration 

<img width="467" alt="image" src="https://github.com/user-attachments/assets/36564107-fef7-429d-9226-9d3073826ea8">
<img width="829" alt="image" src="https://github.com/user-attachments/assets/9bd50793-7ca3-4f29-b3e5-7c62a8e9ec9b">

Reply:
I've recreated migration